versions.yaml

```diff
@@ -2,8 +2,7 @@ supported:
   - channel: '0.37'
     version: 'v0.37.0-rc1'
   - channel: '0.36'
-    version: 'v0.36.0'
-deprecated:
+    version: 'v0.36.0-rc1'
```

Comment:
Can you revert the changes to this file? Looks like a bad merge.

For context this file drives our release process. It shouldn't change in this PR.